zerver/views/realm.py

```diff
@@ -60,6 +63,34 @@ def parse_jitsi_server_url(
 JITSI_SERVER_URL_MAX_LENGTH = 200
 
 
+check_group_setting: Validator[Union[int, Dict[str, List[int]]]] = check_union(
+    [
+        check_int,
+        check_dict_only(
+            [
+                ("direct_members", check_list(check_int)),
+                ("direct_subgroups", check_list(check_int)),
+            ]
+        ),
+    ]
+)
```

Comment:
I was working on a PR for converting this to `typed_endpoint`. I will try to open the PR tommorrow, so that we do not need these changes.